installer/msi/build-msi.ps1

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2018-Present MongoDB Inc.
+<#
+.SYNOPSIS
+    Builds an MSI for the MongoDB ODBC driver.
+.DESCRIPTION
+    .
+.PARAMETER Arch
+    The architecture, x86 or x64.
+#>
+Param(
+  [string]$Arch,
+  [string]$VersionLabel
+)
+
+$ErrorActionPreference = 'Stop'
+
+$ProjectName = "Atlas SQL ODBC"
+$sourceDir = pwd
+$resourceDir = pwd
+$binDir = pwd
+$objDIr = ".\objs\"
+$WixPath = "C:\wixtools\bin\"
+# for local building, most installations will be in the directory below
+#$WixPath = "C:\Program Files (x86)\WiX Toolset v3.11\bin"
+$wixUiExt = "$WixPath\WixUIExtension.dll"
+
+if ($VersionLabel -eq "snapshot") {
+    # We use 0.1.0 as the default for a snapshot because Windows seems to think a newer version is installed if
+    # 0.0.0 is used even when nothing is installed.
+    $VersionLabel = "0.1.0"
+}
+if (-not ($VersionLabel -match "(\d\.\d).*")) {
+    throw "invalid version specified: $VersionLabel"
+}
+$version = $matches[1]
+
+# upgrade code needs to change everytime we
+# rev the minor version (1.0 -> 1.1). That way, we
+# will allow multiple minor versions to be installed
+# side-by-side.
+if ([double]$version -gt 0.1) {
```

Comment:
Created [SQL-1323](https://jira.mongodb.org/browse/SQL-1323) to try to avoid the manual step of generating and updating a new GUID when a minor/major update occurs.